### PR TITLE
Add Dockerfile and replace bower with yarn

### PR DESCRIPTION
**Pull request by Bobfrat**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+COPY ./web/ /web/
+RUN yarn global add grunt-cli && \
+    yarn install && \
+    grunt
+
+FROM python:3.6
+
+RUN mkdir -p /glider-dac-status
+RUN mkdir -p /glider-dac-status/logs
+COPY app.py config.yml flask_environments.py manage.py /glider-dac-status/
```
Comment (benjwadams) [inline Dockerfile]: Should be able to copy all of these in one step.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+COPY ./web/ /web/
+RUN yarn global add grunt-cli && \
+    yarn install && \
+    grunt
+
+FROM python:3.6
+
+RUN mkdir -p /glider-dac-status
+RUN mkdir -p /glider-dac-status/logs
```
Comment (benjwadams) [inline Dockerfile]: `mkdir -p` will create any nonexistent parent directories if they don't exist, so the call to `mkdir -p /glider-dac-status` is redundant.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+COPY status /glider-dac-status/status
+COPY navo /glider-dac-status/navo
+COPY requirements/requirements.txt /requirements.txt
+
+WORKDIR /glider-dac-status
+
+RUN apt-get update && \
+    apt-get install -y python3-netcdf4 && \
+    pip install --no-cache cython gunicorn && \
+    pip install --no-cache -r /requirements.txt && \
```
Comment (benjwadams) [inline Dockerfile]: Add `rm -rf /var/lib/apt/lists/*` to clear out the `apt` cache, which assists in keeping image size down.

**Comment by Bobfrat** [comment]
Thanks Ben, I updated where I could

**Pull request merged**